app/build.gradle

```diff
@@ -34,32 +38,47 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            resValue("bool", "FIREBASE_ANALYTICS_DEACTIVATED", "false")
+        }
+        debug {
+            resValue("bool", "FIREBASE_ANALYTICS_DEACTIVATED", "true")
         }
     }
+
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
```

Comment:
Why roll back to Java1.8? Could this cause issues in the future?

Reply:
Oops, will add a comment clarifying why. One of the libraries used requires the version to be at 1.8 but it shouldn't cause any problems!